src/validation.cpp

```diff
@@ -3093,16 +3093,26 @@ void CChainState::ResetBlockFailureFlags(CBlockIndex *pindex) {
         }
         it++;
     }
+	
+	CBlockIndex *pindexFirstInvalid = nullptr;
```

Comment:
Changing tab to spaces:

```suggestion
   CBlockIndex *pindexFirstInvalid = nullptr;
```